crates/common/src/add_order.rs

```diff
@@ -232,10 +233,31 @@ impl AddOrderArgs {
     }
 }
 
+pub fn random_u256() -> U256 {
```

Comment:
Why do we need this function at all? instead of just calling `rand::random()` directly?

if we're intending for this function to be used across all our crates consistently this is the wrong place for it to live, if we aren't then i would say to delete it